src/Dots/Dots.jsx

```diff
@@ -3,33 +3,36 @@ import React, { Component, PropTypes } from 'react'
 import Dot from './Dot'
 import dynStyle from './style'
 
-const maxDots = 3
-const maxOpacity = 0.6
-const minOpacity = 0.1
-const timeout = 1000
+const MAX_DOTS = 3
+const MAX_OPACITY = 0.6
+const MIN_OPACITY = 0.1
+const TIMEOUT = 1000
 
 class Dots extends Component {
   constructor(props, context) {
     super(props, context)
 
     this.state = {
-      opacities: Array.from(Array(maxDots)).map(() => minOpacity),
+      opacities: Array.from(Array(MAX_DOTS)).map(() => MIN_OPACITY),
```

Comment:
I prefer this package to have as little dependency number as possible.
So I try to not use lodash.